.github/workflows/node-ci.yml

```diff
@@ -31,8 +31,9 @@ jobs:
               id: match
               with:
                   changed-files: ${{ steps.changed.outputs.files }}
-                  files: packages/ # Only look for changes in packages
-                  globs: "!(**/__tests__/*), !(**/__stories__/*), !(**/dist/*)" # Ignore test files
+                  files: "packages/, config/build/" # Only look for changes in packages, build
```

Comment:
I'm tempted to just not specify the packages we monitor.  The `changeset-per-package` will get the workspace info and then check that each change in a package has a changeset already, right?  So does it hurt letting all files that aren't ignored by the glob through and then letting changeset-per-package figure things out?  I'm probably being paranoid, but if we add another utility package somewhere else in the repo we'd have to remember to update this `files:` key. 🤷‍♂️ 

I'll leave it with you to decide.

Reply:
Files has a default so if we don't provide something it'll be limited to `github/`